frontend/src/components/common/ReleaseNotes/ReleaseNotes.tsx

```diff
@@ -77,30 +103,43 @@ export default function ReleaseNotes() {
             // Calling this after setting the version above, so the release notes have the right version
             // set when they show it.
             if (!!releaseNotes) {
+              releaseNotesRef.current = releaseNotes;
```

Comment:
We should also call `setFetchingRelease(false)` here since it means that we've reached what we told the user we were doing, no need for the popup to be there.

Reply:
fixed